### DISCUSSION TITLE
Password screen not coming if the entered user email does not match with any of the preconfigured IDP domains

### DISCUSSION THREAD

**Discussion by manishchoudhary7496** [sventorben/keycloak-home-idp-discovery#382] [Q&A] [answered]
I want the user to enter username and then the homeidp flow will check if the email matches with any of the configured IDPs domain configuration, if not then what will happen?
As per one of the video tutorial for same, I got to know that password screen should come but in my case the password screen is not coming and i am getting Error "Incorrect username or password".

I am not sure why, can anyone Please help me in this.

I have invested 2 days but no luck.

**Comment by sventorben** [comment, marked as answer]
Please see https://github.com/sventorben/keycloak-home-idp-discovery/issues/381 for the answer.
